airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -122,6 +132,46 @@ public static Source sshWrappedSource() {
     return new SshWrappedSource(new MySqlSource(), JdbcUtils.HOST_LIST_KEY, JdbcUtils.PORT_LIST_KEY);
   }
 
+  private ConnectorSpecification modifySpecForCloud(final ConnectorSpecification originalSpec) {
```

Comment:
nit: rename this `modifiedSpecForCloud` because the current name implies that you're mutating the spec passed as argument